boards/nucleo-f303/include/periph_conf.h

```diff
@@ -101,8 +101,8 @@ static const uart_conf_t uart_config[] = {
     {
         .dev        = USART3,
         .rcc_mask   = RCC_APB1ENR_USART3EN,
-        .rx_pin     = GPIO_PIN(PORT_B, 11),
-        .tx_pin     = GPIO_PIN(PORT_B, 10),
+        .rx_pin     = GPIO_PIN(PORT_C, 11),
+        .tx_pin     = GPIO_PIN(PORT_C, 10),
```

Comment:
OK I misread, it's changing something on nucleo-f303, so I think this should be in a separated PR.